main.js

```diff
@@ -1,14 +1,35 @@
-let person = {
-  name: "Yoda",
-  designation: "Jedi Master ",
-};
-
-function trainJedi(jediWarrion) {
-  if (jediWarrion.name === "Yoda") {
-    console.log("No need! already trained");
+let user = 'John Doe';
+console.log(user);
+
+let student = 'Eduard';
+console.log(student);
+
+let test = 1;
+test = test + 1;
+test += '1'; // 21;
+console.log(test);
+test -= 1; // 20;
+console.log(test); // true
+console.log(Boolean(test));
+
+let result = 1;
+let arr = [2, 3, 5, 8];
+for (let i = 0; i < 4; i++) {
```

Comment:
in the future 
this part **i < 4** will be better to replace **i < arr.length**
in this case if array length changes, your code will be not needed to change